Sources/Apodini/Semantic Model Builder/InterfaceExporter/ConnectionContext.swift

```diff
@@ -160,15 +223,73 @@ class EndpointSpecificConnectionContext<I: InterfaceExporter, H: Handler>: Conne
         }
     }
 
-    override func register<Listener: ObservedListener>(listener: Listener) where Listener.Exporter == I {
+    /// Register a listener that will be notified once an observed object did change in the handler
+    /// that is being used in this connection.
+    public func register<Listener: ObservedListener>(listener: Listener) {
         // register the given listener for notifications on the handler
         for object in collectObservedObjects(from: endpoint.handler) {
             self.observations.append(object.register { [weak self] triggerEvent in
```

Comment:
There is no need to weakify `self` since it is not getting used in the closure anymore

Reply:
Thanks, fixed it!